src/components/HighlightedText/styled/index.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+
+// eslint-disable-next-line import/prefer-default-export
+export const HighlightedContainer = attachThemeAttrs(styled.div)`
```

Comment:
If we aren't adding some styles to the container, we would not create a styled for that